swarm/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Unreleased
+
+- Allow users to opt out of the `NetworkBehaviourEventProcess` 
+mechanism through `#[behaviour(event_process = false)]`. This is
+useful if users want to process all events while polling the
+swarm through `SwarmEvent::Behaviour`.
```

Comment:
I believe that this is a change in `core-derive`, not `swarm`.

Reply:
Yeah true, will update!

Reply:
Done!